app/scripts/react.module.ts

```diff
@@ -4,17 +4,25 @@ import * as ReactGA from 'react-ga';
 
 import {SETTINGS} from './modules/core/config/settings';
 
-import {AuthenticationServiceInject} from './modules/core/authentication/authentication.service';
+// angular services to make importable
+import {AUTHENTICATION_SERVICE, AuthenticationServiceInject} from './modules/core/authentication/authentication.service';
```

Comment:
might as well change this import from `'./modules/core/` to `core/` if it'll work